src/python/library/tritonclient/grpc/aio/__init__.py

```diff
@@ -112,6 +116,16 @@ async def close(self):
         """
         await self._channel.close()
 
+    async def _pre_api_call(self, headers):
```

Comment:
Question: not sure on the need / use case for async plugins - maybe we can discuss more offline.

Reply:
Removed async APIs based on the offline discussion.